src/components/pagination.rs

```diff
@@ -6,6 +6,28 @@ use crate::{
 use yew::prelude::*;
 use yew_hooks::use_click_away;
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum PaginationPosition {
+    Top,
+    Bottom,
+}
+
+impl PaginationPosition {
+    fn menu_classes(&self) -> Classes {
```

Comment:
Not wrong! But for cases like this, we have the `AsClasses` trait. Which takes some value and turns it into classes. In different ways.

All you need to to is to implement the trait for `PaginationPosition`, like so: https://github.com/ctron/patternfly-yew/blob/e884d934db875a2bff60c52c306310989678f207/src/components/panel/mod.rs#L25-L33

And then you can do the following: https://github.com/ctron/patternfly-yew/blob/e884d934db875a2bff60c52c306310989678f207/src/components/panel/mod.rs#L67

Reply:
My last commit is using `AsClasses` as suggested.